homeassistant/components/drop_connect/select.py

```diff
@@ -0,0 +1,96 @@
+"""Support for DROP selects."""
+
+from __future__ import annotations
+
+from collections.abc import Callable, Coroutine
+from dataclasses import dataclass
+import logging
+from typing import Any
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import CONF_DEVICE_TYPE, DEV_HUB, DOMAIN
+from .coordinator import DROPDeviceDataUpdateCoordinator
+from .entity import DROPEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+# Select type constants
+PROTECT_MODE = "protect_mode"
+
+PROTECT_MODE_OPTIONS = ["AWAY", "HOME", "SCHEDULE"]
+
+FLOOD_ICON = "mdi:home-flood"
+
+
+@dataclass(kw_only=True, frozen=True)
+class DROPSelectEntityDescription(SelectEntityDescription):
+    """Describes DROP select entity."""
+
+    value_fn: Callable[[DROPDeviceDataUpdateCoordinator], str | None]
+    set_fn: Callable[[DROPDeviceDataUpdateCoordinator, str], Coroutine[Any, Any, None]]
```

Comment:
See: https://github.com/home-assistant/core/pull/106314. We can use `Awaitable`.

Reply:
Fixed in b259dd4d91e0cbad3a762f33c372769bd3327f98.